service/searcher/service.go

```diff
@@ -78,7 +78,7 @@ func (c *Service) Search(request Request) (*Response, error) {
 	if err != nil {
 		kubernetesSearchErrorTotal.Inc()
 		c.logger.Log("level", "error", "message", "could not get secret", "stack", fmt.Sprintf("%#v", err))
```

Comment:
```suggestion
```

Reply:
^ That would be also my suggestion.

Reply:
I think Joseph needs more **REQUEST CHANGES**.

Reply:
lol, joseph needs to not just insta-merge

shout at me if you want me to remove this log line :D

Reply:
I already mentioned that and wanted to make your life as easy as possible. Please remove the logging there. It does not make sense, even though you may did not add it yourself.